testsuite/pytests/test_spatial/test_connect_layers.py

```diff
@@ -237,6 +278,16 @@ def test_connect_layers_bernoulli_mask(self):
         }
         self._check_connections(conn_spec, 108)
 
+    def test_connect_layers_poisson_mask(self):
+        """Connecting layers with pairwise_poisson and mask"""
+        conn_spec = {
+            "rule": "pairwise_poisson",
+            "pairwise_avg_num_conns": 0.5,
+            "mask": {"rectangular": {"lower_left": [-5.0, -5.0], "upper_right": [0.1, 0.1]}},
+        }
+        conns = self._check_connections(conn_spec, None, return_conns=True)
+        np.testing.assert_allclose(54, len(conns), atol=5)
+
```

Comment:
This test generates exactly the same connectivity pattern as `test_connect_layers_poisson_kernel_mask()` below, just that the test below applies a more stringent test. I suggest to remove this test therefore.
```suggestion
```